apps/api/v2/src/modules/oauth-clients/controllers/oauth-clients/oauth-clients.controller.ts

```diff
@@ -109,6 +116,21 @@ export class OAuthClientsController {
     return { status: SUCCESS_STATUS, data: client };
   }
 
+  @Get("/managed-users/:clientId")
+  @HttpCode(HttpStatus.OK)
+  @Roles([MembershipRole.ADMIN, MembershipRole.OWNER, MembershipRole.MEMBER])
+  @DocsOperation({ description: AUTH_DOCUMENTATION })
+  async getOAuthClientManagedUsersById(
+    @Param("clientId") clientId: string
+  ): Promise<GetOAuthClientManagedUsersResponseDto> {
+    const existingManagedUsers = await this.userRepository.findManagedUsersByOAuthClientId(clientId, 0, 50); // second argument is for offset while third is for limit
```

Comment:
We should have limit and offset as query parameters using "@Query" decorator. Example can be seen in "getBookings" of "https://github.com/calcom/cal.com/blob/1cfd2f705306ccf8c6234e600ae6b74547f80872/apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts" and then remove comment "// second argument is for offset while third is for limit"

Reply:
Hmm, why do we need to add the "getOAuthClientManagedUsersById" endpoint? In "oauth-client-users.controller.ts" we have identical one:

<img width="1009" alt="Screenshot 2024-05-07 at 15 13 03" src="https://github.com/calcom/cal.com/assets/42170848/718a15aa-9de3-47f4-bd68-46a72def956f">